branca/utilities.py

```diff
@@ -119,6 +119,8 @@ def color_brewer(color_code, n=6):
         raise ValueError("The minimum number of colors in a"
                          " ColorBrewer sequential color series is 3")
 
+    assert isinstance(color_code, str), \
```

Comment:
I'll change it. Can you tell me why you prefer that?

Reply:
> Can you tell me why you prefer that?

To have a `ValueError`, which is more meaningful IMO, than a generic assert error.

Reply:
Makes sense. I mostly use asserts on these type of input checks because it's one less line and power users/in production can disable them, saving processing time.